### ISSUE TITLE
Fix for issue #35 is missing 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dschulten/lambdaj#55]
```
It should look like:

private static Number typedZero(Class<?> numberClass) {
    if (numberClass == Double.class) return 0.0;
    if (numberClass == Float.class) return 0.0f;
    return 0;
}
```

Original issue reported on code.google.com by `renato.c...@gmail.com` on 16 Dec 2010 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
The title of the issue should be: "Fix for issue #35 is missing if clause for 
Float"

```

Original comment by `renato.c...@gmail.com` on 16 Dec 2010 at 10:05

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 2.3.2
```

Original comment by `mario.fu...@gmail.com` on 17 Dec 2010 at 8:38
- Changed state: **Fixed**
